### ISSUE TITLE
Train Test Split are Wrong for Skoda dataset

### ISSUE THREAD

**Issue by JoeZYX** [saif-mahmud/self-attention-HAR#5]
You used the train_test_split from sklearn for Skoda dataset.  But you did not set the shuffle parameter in train_test_split function. It will break all temporal dependencies within the data. The data is reordered.

Then you applied the sliding window function on the reordered data.

**Comment by JoeZYX** [comment]
In the following code, you can see the problem!

def split(data):


    y = data[:, 0]  # .reshape(-1, 1)
    X = np.delete(data, 0, axis=1)

    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.1, random_state=1)
    X_train, X_val, y_train, y_val = train_test_split(X_train, y_train, test_size=0.1, random_state=1)

    return X_train, y_train, X_test, y_test, X_val, y_val

**Comment by JoeZYX** [comment]
I just noticed that you also used this dataloader for another paper.  Could you just fix the code ?
